src/protocol/attribution/accumulate_credit.rs

```diff
@@ -28,24 +31,18 @@ impl AsRef<str> for Step {
     fn as_ref(&self) -> &str {
         match self {
             Self::HelperBitTimesIsTriggerBit => "helper_bit_times_is_trigger_bit",
-            Self::BTimesStopBit => "b_times_stop_bit",
+            Self::BTimesCurrentStopBit => "b_times_current_stop_bit",
             Self::BTimesSuccessorCredit => "b_times_successor_credit",
+            Self::BTimesSuccessorStopBit => "b_times_successor_stop_bit",
         }
     }
 }
 
 /// Accumulation step for Oblivious Attribution protocol.
 #[allow(dead_code)]
-pub struct AccumulateCredit<'a, F: Field> {
-    input: &'a Batch<AttributionInputRow<F>>,
-}
-
-impl<'a, F: Field> AccumulateCredit<'a, F> {
-    #[allow(dead_code)]
-    pub fn new(input: &'a Batch<AttributionInputRow<F>>) -> Self {
-        Self { input }
-    }
+pub struct AccumulateCredit {}
 
+impl AccumulateCredit {
```

Comment:
Honestly, we are eliminating the use of structs for all the other protocols. We should probably just eliminate the struct entirely in this PR.